tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -62,15 +66,18 @@ if [[ $MODE != "aot" ]];then
     python3 -m venv wasi-env && source wasi-env/bin/activate
     python3 -m pip install -r test-runner/requirements.txt
 
-    # Stress test requires max-threads=8 so it's run separately
-    if [[ -e "${THREAD_INTERNAL_TESTS}spawn_stress_test.wasm" ]]; then 
-        ${IWASM_CMD_STRESS} ${THREAD_INTERNAL_TESTS}spawn_stress_test.wasm
-        ret=${PIPESTATUS[0]}
-        if [ "${ret}" -ne 0 ]; then
-            echo "Stress test spawn_stress_test FAILED with code " ${ret}
-            exit_code=${ret}
+    # Stress tests require max-threads=8 so they're executed separately
+    for stress_test in "${STRESS_TESTS[@]}"; do
+        echo "${stress_test}" is a stress test
+        if [[ -e "${THREAD_INTERNAL_TESTS}${stress_test}" ]]; then
```

Comment:
when is this condition not true?

Reply:
We're just not executing the test as the wasm doesn't exist. 

we can just move `echo` inside the if

Reply:
if you don't want to run stress tests for any reasons , the fastest way for you is just to delete .wasm files.
You don't have any other easy ways to avoid running it, so until you're able to disable them explicitly (e.g. until test-runner and refactoring test running system in wamr ) I'd probably leave an opportunity to avoid running stress tests. 

But generally I don't mind and can delete this line